Bug Reporting/ARKEmailBugReporter.h

```diff
@@ -40,6 +51,9 @@ NS_ASSUME_NONNULL_BEGIN
 /// The email body that will be presented to the user when they compose a report.
 @property (nonatomic, copy) NSString *prefilledEmailBody;
 
+/// The email body delegate, responsible for providing key/value pairs to include in the bug report at the time the bug is filed.
+@property (nullable, nonatomic) id <ARKEmailBugReporterEmailBodyAdditionsDelegate> emailBodyAdditionsDelegate;
```

Comment:
because there's the possibility of a retain cycle it should be weak. Good catch!